### DISCUSSION TITLE
i can’t  find the pictures in the iPhone’s album after pushing them to DCIM/100APPLE

### DISCUSSION THREAD

**Discussion by lyt9759** [doronz88/pymobiledevice3#1213] [Q&A]
I can see the pictures in the iPhone’s file management system, and they are named in the format IMG_0xxx.JPEG. Restarting the album or the phone does not solve the problem.

anyone know the reason?

**Comment by erikshe2003** [comment]
You need to delete /PhotoData/Photos.sqlite and restart the device and wait for the system to rebuild the album index~ However, Android devices can manually update the album by commanding f"adb shell am broadcast -a android.intent.action.MEDIA_SCANNER_SCAN_FILE -d file://{remote_path}", Don't know if iOS devices have the same implementation...

**Comment by doronz88** [comment]
I think the proper way to achieve this would be via the `mobilesync` lockdown service, but I haven't researched it

**Reply by e790a8** [reply]
Is there any way to push pictures to the album now?
